src/UvPipeline.cpp

```
@@ -0,0 +1,366 @@
+    std::iota(order.begin(), order.end(), 0);
+    std::sort(order.begin(), order.end(), [&](size_t a, size_t b) {
+        return boxes[a].xmin < boxes[b].xmin;
+    });
+
+    int overlapCount = 0;
+    for (size_t oi = 0; oi < order.size(); ++oi) {
+        const size_t i = order[oi];
+        const TriBox& a = boxes[i];
+        for (size_t oj = oi + 1; oj < order.size(); ++oj) {
+            const size_t j = order[oj];
+            if (boxes[j].xmin > a.xmax)
```

Cppcheck (2.21.0)

[error] There is an unknown macro here somewhere. Configuration is required. If Q_ENUM is a macro then please configure it.

(unknownMacro)